user_guide_src/source/models/entities.rst

```diff
@@ -376,3 +376,22 @@ attribute to check::
 
 Or to check the whole entity for changed values omit the parameter:
     $user->hasChanged();            // true
+Checking for Existence
+-------------------------------
+
+You can check if an Entity exists before and after its created::
+
+    // Find
+    $user = new User();
+    $user = $userModel->find($id);
+    $user-exists();                 // true
```

Comment:
Dropped part of the arrow! Here (and below) should be `$user->exists();`

Reply:
hope its fine now. Please @MGatner let me know if something is missing or needs to be corrected. ^^

Reply:
Looks good!